wormhole-connect/src/utils/inProgressTxCache.ts

```diff
@@ -3,6 +3,12 @@ import { TransactionLocal } from 'config/types';
 const LOCAL_STORAGE_KEY = 'wormhole-connect:transactions:inprogress';
 const LOCAL_STORAGE_MAX = 3;
 
+// Bigint types cannot be serialized to a string
+// That's why we need to provide a replacer function to handle it separately
+// Please see for details: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/BigInt_not_serializable
+const JSONReplacer = (_, value: any) =>
+  typeof value === 'bigint' ? value.toString() : value;
```

Comment:
does it matter that we don't parse these back into `bigint` later when we read the value?

Reply:
Good point! At the time of `JSON.parse` we do not know which property we have previously converted to string (while calling `JSON.stringify`). We can track this in different ways, but track() function doesn't seem to be complaining about that when pass the receipt object, so it may end up being an unnecessary complication. WDYT?

Reply:
@artursapek `route.resume()` is currently supported for manual txs (bridge and CCTP). That's why I wasn't able to use it for resuming the in-progress tx when user clicks on the widget (that's another work).